Script/AtomicNET/AtomicNET/Core/Profiler.cs

```diff
@@ -12,7 +12,7 @@ public partial class Profiler : AObject
                                  [CallerLineNumber] int line = 0)
         {
 #if ATOMIC_PROFILING
-            var profiler = GetSubsystem<Profiler>();
+            var profiler = AtomicNET.Context.GetProfiler();
```

Comment:
When it is right, it is right, very 🐫 !

![brucelee](https://user-images.githubusercontent.com/376203/27603398-d80737d4-5b2a-11e7-9442-0d67ce02ef46.gif)